changelog.md

```diff
@@ -0,0 +1,16 @@
+# Changelog
+Every important changes will be documented in this file
+Tous les changements importants seront document&eacute;s dabns ce fichier.
+
+The format of this file is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).  
+This project is following the [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.0.1]: Unreleased
+### Modification
+- Renaming from 'jira-api-resource' to 'jira-api-issue-resource'
+
+## [1.0.0] - 2020-01-08
+Initial version that supports
+* Read issues
+* Update issues
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[unordered-list-marker-style] Marker style should be `-`](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4819194)